website/content/docs/ecs/configuration-reference.mdx

```diff
@@ -0,0 +1,120 @@
+---
+layout: docs
+page_title: AWS ECS
+description: >-
+  Configuration Reference for Consul on AWS ECS (Elastic Container Service).
+---
+
+# Configuration Reference
+
+This pages details the configuration options for the JSON config format used
+by the `consul-ecs` binary. This configuration is passed to the `consul-ecs` 
+binary as a string using the `CONSUL_ECS_CONFIG_JSON` environment variable.
```

Comment:
Can we add a link to where this is set in the TF

Reply:
I added a brief TF section here. I actually can't link to individual input variables in the Terraform registry docs (no html `id` attribute).